saleor/order/migrations/0119_orderline_is_gift_card.py

```diff
@@ -7,7 +7,7 @@ def set_is_gift_card_field(apps, schema_editor):
     OrderLine = apps.get_model("order", "OrderLine")
     for line in OrderLine.objects.iterator():
         line.is_gift_card = (
-            line.variant and line.variant.product.product_type.kind == "gift_card"
+            bool(line.variant) and line.variant.product.product_type.kind == "gift_card"
         )
         line.save()
```

Comment:
We are only changing `is_gift_card` field, thus we should do `update_fields=["is_gift_card"]` - faster execution and less load for migration